src/components/Nav/index.tsx

```diff
@@ -92,16 +94,38 @@ const Nav: FC<IProps> = ({ path }) => {
                   variant="ghost"
                   isSecondary
                   px={1.5}
+                  _hover={{
+                    transform: "rotate(30deg)", // Rotate the icon to 30 degrees on hover
+                    color: "blue", // Change color to blue on hover
```

Comment:
Same as with the other button icon
```suggestion
                    color: "primary.hover", // Change color to blue on hover
```